### PR TITLE
Add bounce tracking mitigations command to testdriver

### PR DESCRIPTION
**Pull request by g-ortuno**
Spec PR: privacycg/nav-tracking-mitigations#63

This PR adds the required infrastructure to run bounce tracking mitigations from testdriver. The new command correspond to the WebDriver extension command added by the spec PR above.

### CODE REVIEW

**Comment by g-ortuno** [comment]
@wanderview Hi Ben! Do you think you could do a first pass? Or add someone more appropriate? Thanks!

**Comment by wanderview** [comment]
@WeizhongX I was told you might be a good reviewer for testdriver changes.  Can you help review this PR?

**Review by WeizhongX — APPROVED** [review]
LGTM, assuming you are not going to make more changes on this PR.

**Comment by g-ortuno** [comment]
Hi wpt-core-team, would you mind taking a look at this PR for a bounce tracking mitigations command? Also, I see that the bot added 5 reviewers, let me know if that's expected or if I should be adding just one reviewer first in the future. Thanks!

File: resources/testdriver.js
Comment (foolip) [inline resources/testdriver.js:1077]:
```suggestion
         * https://privacycg.github.io/nav-tracking-mitigations/#run-bounce-tracking-mitigations-command`_
```

File: resources/testdriver.js
Comment (foolip) [inline resources/testdriver.js:1072]: What does deletion algorithm refer to here? https://privacycg.github.io/nav-tracking-mitigations/ only uses that phrase once, in a note in https://privacycg.github.io/nav-tracking-mitigations/#bounce-tracking-mitigations-timers.

The WebDriver command calls https://privacycg.github.io/nav-tracking-mitigations/#bounce-tracking-timer so I guess that's the algorithm.

Could this be documented in terms more like what the spec uses, like "Run bounce tracking mitigations"?

File: resources/testdriver.js
Comment (foolip) [inline resources/testdriver.js:1083]: Please also change this case of "deletion algorithm" if the description above is updated.

**Review by foolip — APPROVED** [review]
This requires core team review because it touches testdriver.js, which often requires an RFC. In this case it doesn't, as this is exposing a WebDriver command.

I left a few comments, but don't need to review again if they're addressed.

**Review by g-ortuno — COMMENTED** [review]
Thanks!

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
Unfortunately, infrastructure/ tests are failing /infrastructure/testdriver/send_keys.html in Firefox. Normally it's easy to say there's no plausible connection, but in this case it's not unthinkable that it's a regression because testdriver.js was modified.

First I'll try closing and reopening this PR to run the tests again. If the test is flaky, it might pass on a second try. In parallel I've rebased https://github.com/web-platform-tests/wpt/pull/28759, which is how I usually check if a failure is already on the master branch. If that fails the same test, then it's not caused by this PR.

**Comment by g-ortuno** [comment]
Thanks for merging!

**Pull request merged**
